wpilibc/athena/include/IterativeRobot.h

```diff
@@ -7,68 +7,26 @@
 
 #pragma once
 
-#include "RobotBase.h"
+#include "IterativeRobotBase.h"
 
 namespace frc {
 
 /**
- * IterativeRobot implements a specific type of Robot Program framework,
- * extending the RobotBase class.
+ * IterativeRobot implements the IterativeRobotBase robot program framework.
  *
  * The IterativeRobot class is intended to be subclassed by a user creating a
  * robot program.
  *
- * This class is intended to implement the "old style" default code, by
- * providing the following functions which are called by the main loop,
- * StartCompetition(), at the appropriate times:
- *
- * RobotInit() -- provide for initialization at robot power-on
- *
- * Init() functions -- each of the following functions is called once when the
- *                     appropriate mode is entered:
- *  - DisabledInit()   -- called only when first disabled
- *  - AutonomousInit() -- called each and every time autonomous is entered from
- *                        another mode
- *  - TeleopInit()     -- called each and every time teleop is entered from
- *                        another mode
- *  - TestInit()       -- called each and every time test is entered from
- *                        another mode
- *
- * Periodic() functions -- each of these functions is called each time a
- *                         new packet is received from the driver station:
- *   - RobotPeriodic()
- *   - DisabledPeriodic()
- *   - AutonomousPeriodic()
- *   - TeleopPeriodic()
- *   - TestPeriodic()
- *
+ * Periodic() functions from the base class are called each time a new packet is
```

Comment:
This is super confusing naming—We have "periodic" as it relates to PeriodicRobot, which means "occurring within a discrete time interval", and "periodic" as it's used here, which just means "happens repeatedly". We should clear up this naming confusion.